packages/manager/modules/pci/src/projects/project/private-networks/private-networks.service.js

```diff
@@ -19,13 +19,19 @@ export default class {
     this.OvhApiCloudProjectNetworkPrivateSubnet = OvhApiCloudProjectNetworkPrivateSubnet;
   }
 
-  getPrivateNetworks(serviceName) {
+  getPrivateNetworks(serviceName, customerRegions = []) {
     return this.$http
       .get(`/cloud/project/${serviceName}/aggregated/network`)
       .then(({ data }) => {
         const privateNetworks = {};
+        const localZones = customerRegions?.filter(({ type }) =>
+          type.includes('localzone'),
+        );
         data.resources.forEach((network) => {
-          if (network.visibility === 'private' && network.vlanId) {
+          if (
+            network.visibility === 'private' &&
+            !localZones?.some((region) => region.name === network.region)
```

Comment:
Won't this check fail if localZones is undefined (only case I see now would be that customerRegions is null)
From what I understand of the code we want to handle the case where network is private and it is not located in a localZone
So I guess if localZones is undefined (meaning we don't have localZones) we should pass this test